libpkg/pkg.h.in

```diff
@@ -340,6 +340,13 @@ typedef enum {
 } mirror_t;
 
 typedef enum {
+	REPO_NONE = 0,
+	REPO_PKG,
+	REPO_DEBIAN,
+
+} type_t;
```

Comment:
I'd strongly recommend to rename public typedef to something prefixed (e.g. `pkg_repo_type_t`).